pontoon/base/signals.py

```diff
@@ -94,8 +94,8 @@ def create_locale_permissions_groups(sender, **kwargs):
         return
 
     try:
-        create_group(instance, 'translators', ['can_translate_locale'], '{} translators'.format(instance.code))
-        create_group(instance, 'managers', ['can_translate_locale', 'can_manage_locale'], '{} managers'.format(instance.code))  # noqa
+        create_group(instance, 'translators', ['can_translate_locale'], instance.code)
+        create_group(instance, 'managers', ['can_translate_locale', 'can_manage_locale'], instance.code)  # noqa
```

Comment:
What this ``# noqa`` for? I guess you can drop it now.

Reply:
The line is too long. I'll fix that.